lib/Sheet.js

```diff
@@ -358,6 +363,8 @@ class Sheet {
                 return new Range(startCell, endCell);
             })
             .case(['number', '*', 'number', '*'], (startRowNumber, startColumnNameOrNumber, endRowNumber, endColumnNameOrNumber) => {
+                indexValidation.validateIndex(startRowNumber, "row");
+                indexValidation.validateIndex(endRowNumber, "row");
```

Comment:
You need to validate both row range AND column range, that is why your tests are failing in Sheet.spec.js.

Reply:
For this case I thought that on line 368 the `cell` method of the `Sheet` would be called, which would in turn call the `cell` method on a specific `Row`, which has index validation for the column number on `lib/Row.js:53`.

I wanted to try to keep the validation lower if possible.

Reply:
I missed that. But I did see that the column validation test was failing, that why I brought it up.